Makefile

```diff
@@ -0,0 +1,25 @@
+THOTH_API ?= http://localhost:8000
+
+.PHONY: \
+	build-api \
+	build-app \
+	run-app \
+	run-api \
+
+all: build-api build-app
+
+run-app: build-app
+	RUST_BACKTRACE=1 cargo run start app
+
+run-api: build-api
+	RUST_BACKTRACE=1 cargo run init
```

Comment:
Don't both the `cargo run` commands rebuild the API already? I think calling `build-app` / `build-api` duplicates this.

Reply:
`cargo run` only builds it if it's not already built, i.e. running `run` after `build` doesn't build it twice.